client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss

```diff
@@ -1,4 +1,5 @@
-@import 'calypso/assets/stylesheets/shared/_loading.scss';
+/* stylelint-disable-next-line scss/at-import-no-partial-leading-underscore */
+@import 'calypso/assets/stylesheets/shared/_loading';
```

Comment:
Optional: this import can probably go. It's for the old loading ellipsis.

Reply:
Just checked this. At first it looks like something that we can remove. But its injecting styles that `LoadingEllipsis` needs to work well. I see this component still used in a few flows..
<img width="1148" alt="Screenshot 2022-09-07 at 17 27 15" src="https://user-images.githubusercontent.com/7000684/188918079-4b6cfb1a-88d7-45ba-9647-d765d6c0b67d.png">